docs/man/sat-status.8.rst

```diff
@@ -174,19 +174,39 @@ Query all types of components:
   | x3000c0s5b0    | Ready | OK   | True    | X86  | River | Sling    |
   | x3000c0s7b0    | Ready | OK   | True    | X86  | River | Sling    |
 
-Possible Values
----------------
+OUTPUT FORMAT
+=============
 
+The following fields are displayed in the output of **sat status**
+
+| *xname*
```

Comment:
The rendered man page looks a little nicer with a blank line between the sentence above and this first field description. That can be achieved with:

```suggestion
|
| *xname*
```